File: GameJamProject/Assets/GameManager.cs
using UnityEngine;
using System.Collections;

public class GameManager : MonoBehaviour 
{
	// 経過時間（フレーム）
	private int m_countFrame;
	// スコア
	private int m_score;

	public GUIText m_scoreGUIText;

	// Use this for initialization
	void Start () 
	{
		// 時間
		m_countFrame = 0;
		// スコア
		m_score = 0;
	}
	
	// Update is called once per frame
	void Update () 
	{
		// 経過時間を更新
		m_countFrame++;
		m_scoreGUIText.text = m_score.ToString ();
	}

	// スコア加算
	void Add_Score(int score)
	{
		m_score += score;
	}
	//* アクセッサ ******************************************
	// スコアの値を取得
	int Get_score()
	{
		return m_score;
	}

	// 時間を取得
	int Get_countFrame()
	{
		return m_countFrame;
	}

}

File: GameJamProject/Assets/GameManager.js
#pragma strict

private var timeLeft:float;
private var Score:int;

function Start ( ) {
	this.timeLeft = 0;
	this.Score = 0;
	
	
}

function Update ( ) {
	this.timeLeft -= 1 / 60;
}

function Get_Frame( )
{
	return 60 - this.timeLeft;
}

function Add_Score( addScore:int )
{
	this.Score += addScore;
}